server/tests/e2e.rs

```diff
@@ -176,6 +184,78 @@ async fn e2e_two_nodes_quote_propagation(logger: Logger) {
     assert_eq!(quote.sci(), &sci);
 }
 
+/// Test that two nodes don't propagate dust quotes to eachother.
```

Comment:
Well, the quote does get propagated here from the 2nd server to the 1st so this comment is misleading (the behavior is fine for the time being)